config/vufind/EDS.ini

```diff
@@ -335,6 +335,17 @@ AU = None
 ; This repeatable setting can be used to filter out values using group codes:
 ;excludeGroup[] = 'URL'
 
+; This section can be used to order items based on their labels.
+[ItemCoreOrder]
+; Note the label of the item field from the EDS API first then the relative order it should be shown in
+;Title         = 0
```

Comment:
The .ini parser is more sensitive about which characters are allowed in a key than in a value. Might it be a good idea to reverse this configuration so the numbers are the keys, just in case there is ever a field label containing a character that is illegal in a key name?

Reply:
Great idea Demian, thank you! 
I've pushed that change in a follow-up commit on this PR. I also set the example first key order value to 1 not 0, so that value is not ignored by later logic and for usability :)